src/page/GroupComplete.tsx

```diff
@@ -0,0 +1,90 @@
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import { Thumbnail } from '../assets';
+import CommonButton from '../common/CommonButton';
+import PageLayout from '../components/PageLayout/PageLayout';
+import { GroupCompleteProps } from '../types/GroupComplte/CompleteType';
+
+const GroupComplete = ({ thumbnailUrl, groupPassword }: GroupCompleteProps) => {
+  const navigate = useNavigate();
+  const baseUrl = window.location.origin; // 생성한 그룹 페이지가 만들어지면 대체 될 예정
+
+  const handleCopyClipBoard = async (text: string) => {
+    try {
+      await navigator.clipboard.writeText(text);
+      alert('클립보드에 링크가 복사되었어요.');
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  const handleGroupPageRedirect = () => {
+    alert('그룹 페이지를 생성하고 유지보수 예정.');
+    navigate('/group-page'); // 그룹 페이지 생성 후 유지보수 예정
+  };
+
+  return (
+    <PageLayout category={'group_create'}>
+      <Title>그룹 생성이 완료되었어요!</Title>
+      <PasswordContainer>
+        <PasswordText>
+          비밀번호 <Password>{groupPassword}</Password>
+        </PasswordText>
+      </PasswordContainer>
+      <ThumbnailContainer>
+        {thumbnailUrl ? (
+          <img src={thumbnailUrl} alt="사용자 썸네일" />
+        ) : (
+          <Thumbnail />
+        )}
+      </ThumbnailContainer>
+      <ButtonContainer>
+        <CommonButton
+          onClick={() => handleCopyClipBoard(`${baseUrl}${location.pathname}`)}
+          category="link_copy"
+        />
+        <CommonButton
+          onClick={handleGroupPageRedirect}
+          category="group_direct"
+          isActive={true}
+        />
+      </ButtonContainer>
+    </PageLayout>
+  );
+};
+
+export default GroupComplete;
+
+const Title = styled.h1`
+  margin-top: 7.4rem;
+  margin-bottom: 2.6rem;
```

Comment:
P5: 아래처럼 정의할 수도 있어요!
```suggestion
  margin: 7.4rem 0 2.6rem;
```

Reply:
수정하겠습니다 감사합니다!